Address follow up comments for the multi include guide (#669)
gruntwork-io/gruntwork-io.github.io: _posts/2021-11-23-how-to-dry-your-ref-arch.adoc

}

inputs = {
  # This input variable is for convenience only and is not defined in the underlying Terraform module.
  # We use _ to mark it as "private" and decrease the likelihood of accidentally using a defined variable here.
  _vpc_id = dependency.vpc.outputs.vpc_id
}
----

You can then access the `_vpc_id` input variable in the child configuration by using an exposed `include` block (under

> the reference, `include.PARENT.inputs._vpc_id`, like so:

Reviewer: ```suggestion
the reference, `include.PARENT.inputs._vpc_id`), like so:
```

  }
}
----

Note that since the `dependency` reference is in the `inputs` attribute, you can only reference it in the child

> configuration's `inputs` attribute. You can not configure a `locals` block in the child that accesses the parent

Reviewer: Super NIT

```suggestion
configuration's `inputs` attribute. You cannot configure a `locals` block in the child that accesses the parent
```

> `inputs` attribute.

Reviewer: ➕ Nice!
